api/src/main/java/com/epam/esm/domain/Tag.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Tag implements Serializable {
+
+    private static final long serialVersionUID = -8742791910266118756L;
+    private Long id;
+    private String name;
+    private Set<Certificate> certificates = new HashSet<>();
```

Comment:
Using `Set` here is with default `Certificate.equals` implementation might not be the best decision.